pkg/scheduler/schedulercache/cache.go

```diff
@@ -131,7 +131,7 @@ func (cache *schedulerCache) AssumePod(pod *v1.Pod) error {
 	cache.mu.Lock()
 	defer cache.mu.Unlock()
 	if _, ok := cache.podStates[key]; ok {
-		return fmt.Errorf("pod %v state wasn't initial but get assumed", key)
+		return fmt.Errorf("Pod %v's state wasn't initialized, so can't be assumed", key)
```

Comment:
I would change this to "Pod %v is not found in the cache, so can't be assumed".

Reply:
Done @bsalamat

Reply:
we should not return error message started with big letter :)